docs/includes/prerequisites.rst

```diff
@@ -0,0 +1,5 @@
+.. Prerequisites for using JDBC connector
+
+Prerequisites
+   - If you are using the JDBC source connector with |CP|, you must :ref:`install and start Confluent Platform <installation>` before using.
+   - If you are using the JDBC source connector with a cluster running Apache Kafka, you must have you have ZooKeeper, Kafka, and Schema Registry running before using.
```

Comment:
Do we distinguish in our CP docs between connector deployments on top of CP or just AK? I thought we always assumed users use Confluent platform.

Reply:
@kkonstantine in general, i agree with you that we should should assume and recommend CP installation as the default path. i think i may have misunderstood this issue at first. so it sounds like someone just needs to rewrite this quickstart to use the confluent CLI (and not add verbiage about running with standalone AK)?

this recently came up re: C3 and using with standalone AK.  we solved by creating 2 separate installation paths: [with CP](https://docs.confluent.io/current/control-center/docs/installation/installing-control-center.html) and [with AK](https://docs.confluent.io/current/control-center/docs/installation/install-apache-kafka.html). however, C3 may be a unique case.

Reply:
@kkonstantine how can i make this work with CP CLI and non-standalone Connect?